### ISSUE TITLE
oneDPL documented decision making process

### ISSUE THREAD

**Issue by rodburns** [uxlfoundation/open-source-working-group#47]

**Comment by vbm23** [comment]
@timmiesmith Projects have initiated describing the RFC Process with Template in their respective projects and would be nice if all projects can align in the naming convention and then can always continually improve to include things that do not fit right now
https://github.com/oneapi-src/oneCCL/tree/rfcs
https://github.com/oneapi-src/oneDNN/tree/rfcs

**Comment by akukanov** [comment]
@rodburns Does this issue assume more than just what #35 already covers?

**Comment by vbm23** [comment]
@akukanov Yes, both are tickets are related and https://github.com/oneapi-src/oneDPL/pull/1878 should suffice once approved and merged

**Comment by akukanov** [comment]
The patch is merged.
